packages/typespec-python/tasks.py

```diff
@@ -223,11 +226,23 @@ def _regenerate(
     )
 
 
+def is_invalid_folder(s: Path, invalid_folders: List[str] = []) -> bool:
+    if "sphere" in str(s):
+        return False
+    invalid_folders = invalid_folders + ["type/union"]
```

Comment:
skip `type/union` test since https://github.com/Azure/typespec-azure/pull/3809. After it merged, we can add it back.